app/src/main/java/com/sopt/dive/ui/screens/ProfileScreen.kt

```diff
@@ -0,0 +1,63 @@
+package com.sopt.dive.ui.screens
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.*
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.sopt.dive.ui.theme.MainPinkBackground
+import com.sopt.dive.viewmodel.UserViewModel
+import androidx.compose.runtime.getValue
+
+@Composable
+fun ProfileScreen(paddingValues: PaddingValues, userViewModel: UserViewModel) {
+    val user by userViewModel.currentUser.collectAsState()
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(paddingValues)
+            .padding(20.dp),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Center
+    ) {
+        Text(
+            text = " Profile",
+            fontSize = 28.sp,
+            fontWeight = FontWeight.Bold,
+            textAlign = TextAlign.Center
+        )
+
+        Spacer(modifier = Modifier.height(20.dp))
+
+        Text(text = "ID: ${user?.id}", fontSize = 20.sp)
+
+        Text(text = "PW: ${user?.pw}", fontSize = 20.sp)
+
+        Text(text = "Name: ${user?.name}", fontSize = 20.sp)
+
+        Text(text = "Email: ${user?.email}", fontSize = 20.sp)
+
+        Text(text = "Age: ${user?.age}", fontSize = 20.sp)
+
+        Spacer(modifier = Modifier.height(40.dp))
+
+        Button(
+            onClick = { userViewModel.logoutUser() },
```

Comment:
이처럼 단순한 함수 호출이라면 람다를 새로 생성하기보다는 함수 참조를 그대로 전달하는 편이 더 좋습니다.
람다는 호출 시마다 람다 객체를 생성하기 때문에 함수 참조에 비해 불필요한 리소스를 더 사용하게 돼요 !!
Ex. userViewModel::logoutUser,
```suggestion
            onClick = userViewModel::logoutUser,
```